build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
 
 group = 'com.synopsys.integration'
 
-version = '9.8.0-SIGQA9-SNAPSHOT'
+version = '9.8.0-SIGQA10-idetect-4407-SNAPSHOT'
```

Comment:
Do we want to revert to 9.8.0-SIGQA#-SNAPSHOT before merging or will that not matter? The changes we did to QA pipelines and how we name the pipeline artifacts based on branch names is a bit fuzzy in my memory

Reply:
AFAIK, it shouldn't matter. Others can confirm.

Reply:
I have resolved the conflict as the merge was blocked.